frontend/app/styles/custom.scss

```diff
@@ -1652,3 +1647,16 @@ div.content-input {
   width: 78%;
   font-size: 15px;
 }
+
+button[type="submit"] {
```

Comment:
Will be new with the new base form component, because of that this is a temporary solution, which works